test-d/browser.test-d.tsx

```diff
@@ -3,7 +3,7 @@ import Honeybadger from '../dist/browser/honeybadger'
 
 Honeybadger.configure({
   debug: false,
-  disabled: true,
+  reportData: true,
```

Comment:
Shouldn't this be `false` to match `disabled: true` ?

Reply:
Ah yeah, good catch. It doesn't really matter technically since it's mainly testing the option's existence and type, but it doesn't hurt to match the typical usage. :)

I'll make this change and then merge, assuming no other issues.

Reply:
Sure. No other issues on my side.